packages/bitcore-wallet-client/src/lib/common/utils.ts

```diff
@@ -369,13 +370,18 @@ export class Utils {
       if (data) {
         recipients[0].data = data;
       }
+      const params = {
+        invoiceID,
+        tag: destinationTag ? Number(destinationTag) : undefined
+      };
       const unsignedTxs = [];
       const isERC20 = tokenAddress && !payProUrl;
       const chain = isERC20 ? 'ERC20' : this.getChain(coin);
       for (let index = 0; index < recipients.length; index++) {
         const rawTx = Transactions.create({
           ...txp,
           ...recipients[index],
+          ...params,
```

Comment:
params and txp both have invoiceId.

if the objetive of params is to transform destinantionTag, maybe this could be refactored to:
```
   const rawTx = Transactions.create({
          ...txp,
          ...recipients[index],
          tag: destinationTag ? Number(destinationTag) : undefined,
          chain,
```
and avoid params.

Reply:
done